src/components/GameItem.js

```diff
@@ -13,12 +26,16 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const type = ItemIcons[this.props.type];
+
+
 
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
+      <section>
+      <div className={`game-item ${this.state.cssType}`} style={itemStyle}>
+        <img src={type} alt="Item" className="icon-item" onClick={this.markClickedClickHandler}></img>
```

Comment:
To get a better chance of the click registering, I would put the `onClick` in the div tag rather than the img tag.